composer.json

```diff
@@ -36,9 +36,6 @@
     "autoload": {
         "psr-0": {"IsoCodes": "src/"}
     },
-    "config": {
-        "bin-dir": "bin"
```

Comment:
Didn't see the `Makefile`, will take a look.

BTW, why `make` is not use on Travis? :-)